Sources/snowman/main.swift

```diff
@@ -2,11 +2,17 @@ import SnowManLogic
 import SnowManDictionary
 
 let dictionaryManager = DictionaryManager(words: SnowManDictionary.words) 
-let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
 let word = dictionaryManager.getRandomWord()
 
 let io = IO()
+
 let view = View(io: io)
+
+let maxNumberOfGuesses = 10
+view.promptNumberOfGuesses()
+let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
This works...but, it feels weird to have it hanging out in `main`.  Can you think of somewhere more appropriate for it?